blog_module/views.py

```diff
@@ -0,0 +1,12 @@
+from django.shortcuts import render
+
+from . import models
+
+def index(request):
+    all_blogs = models.Blog.objects.all()
+    return render(request, 'index.html', {'all_blogs': all_blogs})
```

Comment:
Another way to write this is to create a context dict. It's easier to extend and read

```python
...
context = {
   'all_blogs': all_blogs,
   # later more context can  be added here
}
 return render(request, 'index.html', context)
```